src/components/concept/_semgrep-scan.mdx

```diff
@@ -4,4 +4,4 @@ Introduction to what Semgrep is and how it works.
 
 -->
 
-Semgrep uses **rules** to scan code. Matches found based on those rules are called **findings**. A Semgrep rule encapsulates pattern-matching logic and data-flow analysis used to find vulnerabilities such as code violations, security issues, or outdated libraries.
+ Semgrep Code uses **rules**, which encapsulate pattern matching logic and data flow analysis, to scan your code and find issues such as code violations, security issues, and the use of outdated or vulnerable libraries. Semgrep generates and reports to you **findings** whenever it finds code that matches the patterns defined by rules.
```

Comment:
```suggestion
 Semgrep Code uses **rules**, which encapsulate pattern matching logic and data flow analysis, to scan your code and find issues such as code violations, security issues, and the use of outdated or vulnerable libraries. Semgrep generates and reports **findings** to you whenever it finds code that matches the patterns defined by rules.
```

Simplified the English to make it easier for folks to follow. The old text says "...to you findings" - basically I clarified the indirect/direct object.